data/kubernetes-operator-published-branches.yaml

```diff
@@ -1,17 +1,20 @@
 version:
   published:
+    - '0.10'
     - '0.9'
     - '0.8'
   active:
+    - '0.10'
     - '0.9'
     - '0.8'
-  stable: '0.8'
-  upcoming: '0.9'
+  stable: '0.9'
+  upcoming: '0.10'
 git:
   branches:
-    manual: 'v0.8'
+    manual: 'v0.9'
     published:
       - 'master'
+      - '0.9'
```

Comment:
This must be the actual GitHub branch, `v0.9`

Reply:
Agreed. Fixed.